quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/LoggingPropertyMappers.java

```diff
@@ -2,23 +2,38 @@
 
 import static org.keycloak.quarkus.runtime.integration.QuarkusPlatform.addInitializationException;
 
+import java.io.File;
+import java.util.List;
 import java.util.Locale;
+import java.util.Objects;
 import java.util.function.BiFunction;
 import java.util.logging.Level;
 
 import org.jboss.logmanager.LogContext;
+import org.keycloak.quarkus.runtime.Environment;
 import org.keycloak.quarkus.runtime.Messages;
 
 import io.smallrye.config.ConfigSourceInterceptorContext;
 
 final class LoggingPropertyMappers {
 
     private static final String DEFAULT_LOG_LEVEL = "info";
+    private static final String DEFAULT_LOG_HANDLER = "console";
+    private static final String DEFAULT_LOG_FILENAME = "keycloak.log";
+    private static final String DEFAULT_LOG_PATH = "data/log/"+DEFAULT_LOG_FILENAME;
+    private static final List<String> AVAILABLE_LOG_HANDLERS = List.of(DEFAULT_LOG_HANDLER,"file");
+    private static final String DEFAULT_CONSOLE_OUTPUT = "plain";
 
     private LoggingPropertyMappers(){}
 
     public static PropertyMapper[] getMappers() {
         return new PropertyMapper[] {
+                builder().from("log")
+                        .defaultValue(DEFAULT_LOG_HANDLER)
+                        .description("Enable one or more log handlers in a comma-separated list. Available log handlers are: " + String.join(",", AVAILABLE_LOG_HANDLERS))
+                        .paramLabel("<handler>")
+                        .expectedValues("console","file","console,file","file,console")
```

Comment:
actually not, it's either "console" alone, "file" alone, or a combination of these two. This could get messy when we'd have more than 2 options, but for now this leads to the server not even starting with a wrong combination entered from CLI. 

Problem adding it in the transformer instead was for me that the `throwinitializationexception` threw the error more than once, i think because it's a `mapFrom` config property. Also I like to "fail fast" here, now when you do e.g. `kc.sh log=` or `kc.sh log=foo` or `kc.sh log=console,foo` quarkus would not even start but an error is shown instead.

In the future we should really rely on types in the cli, and not handle everything as string. when you'd have a `List<KcLogConfig` as type, we could add the arity for picocli for exactly this type, and also the split. (arity 1...*, split=",") and make this more convenient. 

See also: 
https://github.com/DGuhr/keycloak/tree/PoC_Typed_Converters_For_CLI_Values
https://picocli.info/#_arity
https://picocli.info/#_split_regex